Vic2ToHoI4/Source/OutHoi4/Decisions/OutDecision.cpp

```diff
@@ -19,7 +19,7 @@ std::ostream& HoI4::operator<<(std::ostream& outStream, const decision& outDecis
 	}
 	if (!outDecision.highlightStates.empty())
 	{
-		outStream << "\n\t\thighlight_states " << outDecision.highlightStates << "\n";
+		outStream << "\n\t\thighlight_state_targets " << outDecision.highlightStates << "\n";
```

Comment:
if you changed output keyword, do you wish to dump input keyword? Parsing 1.8 input will produce 1.9 output with other problems..

Reply:
Yeah, dumped the whole 1.8 way of doing things. I'm not that interested in converting to old HoI4.